### PR TITLE
Tags section

### PR DESCRIPTION
**Pull request by mrthegreensky**
Major changes include: changing layout for tags and addition of tags to expense claims.

Change of layout for tags now uses the tag view configurator.

Tags are now able to be:
- [x] Added to expense claims
- [x] Deleted from expense claims
- [x] Edited in expense claims

### CODE REVIEW

File: Team9Project/res/menu/add_tag_to_claim.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<menu xmlns:android="http://schemas.android.com/apk/res/android" >
+    <item android:id="@+id/action_add_tag_to_claim_string" android:alphabeticShortcut="@string/action_add_tag_to_claim" android:title="@string/action_add_tag_to_claim"></item>
```
Comment (indragiek) [inline Team9Project/res/menu/add_tag_to_claim.xml]: Should be reindented in Eclipse (splitting attributes on multiple lines) for readability

File: Team9Project/res/menu/replace_tag.xml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<menu xmlns:android="http://schemas.android.com/apk/res/android" >
+    <item android:id="@+id/action_replace_tag" android:alphabeticShortcut="@string/action_replace_tag" android:title="@string/action_replace_tag"></item>
```
Comment (indragiek) [inline Team9Project/res/menu/replace_tag.xml]: Same here

File: Team9Project/src/com/indragie/cmput301as1/TagEditToClaimActivity.java [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.view.ActionMode;
+import android.view.Menu;
+import android.view.MenuItem;
+import android.view.View;
+import android.widget.AdapterView;
+
+public class TagEditToClaimActivity extends TagAddToClaimActivity {
```
Comment (indragiek) [inline Team9Project/src/com/indragie/cmput301as1/TagEditToClaimActivity.java:26]: It seems like there's quite a lot of duplicated code between this and `TagAddToClaimActivity`. Is there any way to reduce the duplication?

**Comment by indragiek** [comment]
Nice :+1: There's just a few minor things to fix. Also, GitHub is saying this can't be auto merged so you'll have to merge master into this branch and fix conflicts.

File: Team9Project/src/com/indragie/cmput301as1/TagEditToClaimActivity.java (hunk 3 above)
Comment (mrthegreensky) [inline Team9Project/src/com/indragie/cmput301as1/TagEditToClaimActivity.java:26]: Actually, I think there is. Thanks for pointing that out!

**Comment by williums** [comment]
Looks good!

**Comment by indragiek** [comment]
Changes look good, just need to fix conflicts before being able to merge.

**Pull request merged**